libraries/AP_Relay/AP_Relay.h

```diff
@@ -50,6 +50,7 @@ class AP_Relay {
 
     AP_Int8 _pin[AP_RELAY_NUM_RELAYS];
     AP_Int8 _default;
+    AP_Int8 _pin_states;
```

Comment:
```suggestion
    uint8_t _pin_states;
```

Reply:
whats the difference? all other variables in this are AP_Int8?

Reply:
AP_Int8 is a parameter, uint8_t is just a variable

Reply:
thanks...got it